package.json

```diff
@@ -45,5 +45,7 @@
     "react": "*",
     "react-native": "*"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@babel/runtime": "^7.20.7"
```

Comment:
Why do we added this library?

Reply:
I was getting some errors, since it was suggested when searching for the error, I am not familiar with javascipt could there be an issue with that?

Reply:
I tested again and it works without this time, so I removed it